### PR TITLE
Add Limited Write role docs

### PR DESCRIPTION
**Pull request by stefanvanburen**
Again, just queueing this up for now.

### CODE REVIEW

File: docs/bsr/user-management.md [hunk 1]
```diff
@@ -64,6 +64,10 @@ When computing the role on a resource, the highest role takes precedence. For ex
   -  Creating tags
   -  Updating template versions and plugins
 
+### Limited Write {#resource-limited-write}
```
Comment (cyinma) [inline docs/bsr/user-management.md:67]: lets also mention that this resource role only apply to repository currently

**Comment by stefanvanburen** [comment]
We're going to hold this one for #182 being ready to go

**Comment by stefanvanburen** [comment]
Looks like we merged the drafts docs into #194, so this is just waiting for that one to get merged now.

**Review by cyinma — APPROVED** [review]

**Comment by stefanvanburen** [comment]
This is now live on buf.build, merging!:

image

**Pull request merged**